ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -621,16 +607,27 @@ def ConvTranspose(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]
 
 def Pad(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Add padding to the input tensor."""
-    pads = onnx_node.get_attribute_value('pads')
+    paddings = onnx_node.get_attribute_value('paddings')
+    if paddings is None:
+        raise ValueError('Pad node (s%): paddings attribute is required.', onnx_node.name)
+
     constant = 'constant'
     mode = onnx_node.get_attribute_value('mode', constant)  # 'constant', 'reflect' or 'edge'
-    value = onnx_node.get_attribute_value('value', 0)
+    value = onnx_node.get_attribute_value('value', 0.)
+
+    if len(paddings) != 2 * len(ng_inputs[0].shape):
+        raise ValueError('Pad node (%s): \'paddings rank (%d) should be double of input tensor '
+                         'rank (%d).', onnx_node.name, len(paddings), len(ng_inputs[0].shape))
 
+    # Operator set version >= 2
```

Comment:
If this is for Operator set version >= 2, you should remove this from code for opset 1